package.json

```diff
@@ -42,13 +42,15 @@
     "@fastify/cors": "7.0.0",
     "@fastify/helmet": "8.0.0",
     "@fastify/secure-session": "4.0.0",
+    "@graasp/sdk": "github:graasp/graasp-utils#15/testUtils",
```

Comment:
I think we should rename the Github-repo hosting this new `@graasp/sdk` in order to avoid confusion.